bin/dev-server.js

```diff
@@ -0,0 +1,27 @@
+const fs = require("fs");
+const path = require("path");
+const serveIndex = require("serve-index");
+const express = require("express");
+
+const toolbox = require("../node_modules/devtools-local-toolbox/index");
```

Comment:
I guess the main thing was referencing `node_modules` directly though: you should be able to do `require("devtools-local-toolbox/index")` right?